src/library/Ora/ReadModel/Task.php

```diff
@@ -212,4 +217,43 @@ private function getMembersShare() {
 		}
 		return $rv;
 	}
+
+    public function getResourceId(){
+    	return get_class($this);
+    }
+	
+    public function getReadableMembers(){
```

Comment:
Anche qui, i metodi getReadable* sono tutti legati all'implementazione di un'interfaccia Readable* . Invece il metodo getResourceId è necessario in quanto la classe implementa l'interfaccia ResourceInterface